en/docs/learn/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -34,329 +34,323 @@ The following concepts and artifacts are used in this tutorial:
+7. Click **Submit**.
 
-    ```bash
-    {MI_HOST}  :  localhost
-    {MI_PORT}  :  8290
-    {MI_URL}   :  localhost:8290
-    ```
+    Now let's add a **Respond** mediator at the end of the in sequence to send the response message from the healthcare service back to the client.
 
-    !!! Tip
-        See the instructions on [injecting environment variables to the embedded Micro Integrator]({{base_path}}/develop/using-embedded-micro-integrator/#injecting-environment-variables-to-embedded-micro-integrator).
+8. Click on the **+** icon in the sequence to add a Respond mediator after the Call mediator.
+
+    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-respond.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-respond.png" alt="add respond" width="80%"></a>
+
+9. From the **Palette**, select **Respond** mediator under the **Mediators** > **Generic** section.
 
-### Step 4 - Package the artifacts
+    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/respond-mediator.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/respond-mediator.png" alt="respond mediator" width="30%"></a>
 
-Package the artifacts in your composite exporter module to be able to deploy the artifacts in the server.
+10. Click **Submit**.
 
-1. Open the `pom.xml` file of the **ServiceCatalogSampleCompositeExporter** module.
-2. Ensure that the following artifacts are selected in the POM file.
+You have successfully created all the artifacts that are required for sending a request through the Micro Integrator to the back-end service.
 
-    - `HealthcareAPI`
-    - `QueryDoctorEP`
+<a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/integration-sequence.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/integration-sequence.png" alt="integration sequence" width="80%"></a>
 
-3. By default, the `Publish to Service Catalog` checkbox is enabled. If not, select the checkbox in the wizard so that it will include metadata files of the selected artifacts. 
+### Step 3: Configure the Micro Integrator
 
-4. Save the changes.
+!!! info "Prerequisites"
+    Before you begin, install Micro Integrator on your machine:
 
-### Step 5 - Start the API Manager runtime
+    1. Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, provide necessary details, and then click **Zip Archive** to download the Micro Integrator distribution as a ZIP file.
+    
+    2. Extract the ZIP file. The extracted folder will be referred as the `<MI_HOME>` folder.
+
+The Micro Integrator contains a client application, which automatically publishes artifacts to the **Service Catalog** in the **API Publisher** portal. 
```

Comment:
Do we have to mention about the client application here, as it is an internal detail

Reply:
Fixed with [dcbc2d1](https://github.com/wso2/docs-mi/pull/984/commits/dcbc2d1cb3e0dd38f951300393aca407744f0acd)